easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2023.07-iimkl-2023a.eb

```
@@ -0,0 +1,114 @@
+easyblock = 'PythonBundle'
+
+name = 'SciPy-bundle'
+version = '2023.07'
+
+homepage = 'https://python.org/'
+description = "Bundle of Python packages for scientific software"
+
+toolchain = {'name': 'iimkl', 'version': '2023a'}
+toolchainopts = {'pic': True, 'lowopt': True}
```

Let's try enabling `strict` here too, so different compiler options 

```suggestion
toolchainopts = {'pic': True, 'lowopt': True, 'strict': True}
```

Could also try without `lowopt` (so using default `-O2` instead of `-O1` that's implied by `lowopt`).

To figure out if it's' due to a compiler bug, we could also try with `noopt`